.gitignore

```diff
@@ -6,6 +6,7 @@ vendor/
 /phpstan.neon
 /phpunit.xml
 
+.idea/
```

Comment:
PHPStorm folder, wasn't sure what the convention was regarding editor specific ignores so I left it here

Reply:
Please remove this/.